### PR TITLE
Update AB3's GUI for HireLah

### PR DESCRIPTION
**Pull request by likeabowx**
Updated the GUI as follows:
1. Update title name and icon 
2. Add labels for gender and age in person card
3. Add 'Currently Displaying' label to indicate the type of data being shown

image

### CODE REVIEW

File: src/main/resources/view/DarkTheme.css [hunk 1]
```diff
@@ -127,11 +127,22 @@
...
 
 .cell_small_label {
-    -fx-font-family: "Segoe UI";
+    -fx-font-family: "Segoe UI Semibold";
     -fx-font-size: 13px;
     -fx-text-fill: #010504;
 }
 
+.currently_displaying {
+    -fx-font-family: "Segoe UI Semibold";
```
Comment (goalfix) [inline src/main/resources/view/DarkTheme.css:136]: Would it be possible to change the font family to this instead?
```
* {
   -fx-font-family: 'serif'
}
```

Rationale: My 2015 mac is unable to read the Segoe UI unfortunately.

File: src/main/resources/view/DarkTheme.css [hunk 2]
```diff
@@ -127,11 +127,22 @@
 }
 
 .cell_small_label {
-    -fx-font-family: "Segoe UI";
+    -fx-font-family: "Segoe UI Semibold";
```
Comment (goalfix) [inline src/main/resources/view/DarkTheme.css:130]: Similar to below

**Review by goalfix — CHANGES_REQUESTED** [review]
Minor changes requested over font family. LGTM otherwise!

File: src/main/resources/view/DarkTheme.css (hunk 1 above)
Comment (likeabowx) [reply src/main/resources/view/DarkTheme.css:140]: Changed the font to sans-serif

**Review by goalfix — APPROVED** [review]
LGTM!

**Pull request merged**